components/ColumnToggle.tsx

```
@@ -3,19 +3,40 @@ import { ColumnVisibility } from "@/types/exam";
 }) => {
   return (
     <div className="mb-6 flex flex-col items-center gap-4 text-sm select-none">
       <div className="flex items-center gap-2 text-primary-text font-medium">
-        <svg className="h-5 w-5 text-primary-text" fill="none" stroke="currentColor" viewBox="0 0 24 24">
-          <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M9 17V7m0 10a2 2 0 01-2 2H5a2 2 0 01-2-2V7a2 2 0 012-2h2a2 2 0 012 2m0 10a2 2 0 002 2h2a2 2 0 002-2M9 7a2 2 0 012-2h2a2 2 0 012 2m0 10V7m0 10a2 2 0 002 2h2a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v10a2 2 0 002 2h2a2 2 0 002-2z" />
+        <svg
+          className="h-5 w-5 text-primary-text"
+          fill="none"
+          stroke="currentColor"
+          viewBox="0 0 24 24"
+        >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)